### ISSUE TITLE
Contrast accessibility issues

### ISSUE THREAD

**Issue by struthj** [SmarterApp/AP_ItemSampler#6]
Contrast 
Text and images of text with insufficient contrast can be less readable, especially for users with low vision. WCAG 2.0 Level AA requires a minimum contrast ratio of 4.5:1 for normal text and 3:1 for large text – 14 point (18.6 pixels) and bold or larger, or 18 point (24 pixels) or larger. WebAIM's contrast checker can generate this contrast ratio and give immediate feedback as to whether or not the contrast is sufficient. Simply enter a foreground and background color to see the results. Additionally, color alone should never be used as the only means to convey important information, text should be readable, and pages responsive to changes in text size. Specific instances of insufficient contrast are provided in this section, and are noted elsewhere in the report. 

~~- [ ] On some pages (e.g., About Test Items), the white text on the test item number `<h2> `element (`class="questionNumber"`) has very low 2.2:1 contrast with the light-blue (#6cb9c6) background. Severity 2~~
EDIT this is an Iframe issue

- [x] Some links and buttons have only a very subtle change in contrast on mouse hover. Minimum WCAG 2.0 requirements are met, but accessibility could be still be improved by improved by making the underline style that is provided on keyboard focus available on mouse hover as well. This can be accomplished by adding hover to the existing CSS (e.g.`,a:focus, a:hover {... `}). Severity 4 
[Example image of hover state colors](http://imgur.com/a/zSULp)
